readthedocs/doc_builder/python_environments.py

```diff
@@ -193,12 +193,12 @@ def install_core_requirements(self):
 
         # Use conda for requirements it packages
         requirements = [
-            'sphinx==1.3.5',
+            'sphinx==1.5.3',
```

Comment:
Wouldn't it be nicer to pin Sphinx because it's so vital that bumping the version should be challenged every time by a deliberate action (PR) ?